### PR TITLE
improvement(apiv2): use the OTP-UI conversion utility for legacy format

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
**Description:**
Uses the OTP-UI utility function to convert new OTP response into old for use in the app.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Comment by daniel-heppner-ibigroup** [comment]
Didn't realize the Percy tests broke on this.

File: lib/actions/apiV2.js [hunk 1]
```diff
@@ -962,7 +922,7 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
               const withCollapsedShortNames =
                 response.data?.plan?.itineraries?.map((itin) => ({
                   ...itin,
-                  legs: itin.legs?.map(processLeg)
+                  legs: itin.legs?.map(convertGraphQLResponseToLegacy)
```
Comment (binh-dam-ibigroup) [inline lib/actions/apiV2.js:925]: This is great. Could you also update the dependencies to use core-utils 11.0.2 when it gets published?

Comment (daniel-heppner-ibigroup) [reply lib/actions/apiV2.js:925]: Sure, makes sense to roll that into this

**Review by binh-dam-ibigroup — APPROVED** [review]
Seems to work!

**Pull request merged**